README.rst

```diff
@@ -34,15 +40,28 @@ See https://learningequality.org/kolibri/ for more info.
 How can I use it?
 -----------------
 
-Kolibri is under active development and is not yet ready to be used. In the meantime, take a look at `KA-Lite <https://learningequality.org/kolibri/>`_, Kolibri's predecessor which is already deployed around the world.
+Kolibri is under active development and is not yet ready to be used in real
+deployments.
+
+**We expect a public beta to be available during second half of 2017.**
+
+In the meantime, take a look at
+`KA-Lite <https://learningequality.org/kolibri/>`_, Kolibri's predecessor which
+is already deployed around the world.
 
 
 How can I contribute?
 ---------------------
 
 .. warning::
-  We welcome new contributors but since **Kolibri** is still in development, the API is not yet completely ready to integrate external plugins. Please start by:
+
+  *We welcome new contributors but since* ***Kolibri*** *is still in*
+  *development, the APIs are subject to change, and upgrade paths are not*
+  *guaranteed for all releases. We do not recommend deploying Kolibri before*
+  *the first public betas are available (for the very most urgent needs).*
```

Comment:
But do we recommend deploying the first public beta? :) Okay, I can leave it out - of course things that we explicitly do not recommend, do not imply that we then recommend everything else :)